### ISSUE TITLE
"Wrap Reverse" and "Flex End" do not mesh well when the content is larger than the flexbox

### ISSUE THREAD

**Issue by soumya92** [google/flexbox-layout#250]
The inconsistent behaviour centres around the "Align content" attribute. If set to "Flex start", then the items are aligned to the bottom, as expected:
![start-fits](https://cloud.githubusercontent.com/assets/507289/24883771/421f9944-1dfb-11e7-9907-3a252a0e3473.png)

When there are more items than can fit, the items towards the top are clipped, also as expected:
![start-doesnot-fit](https://cloud.githubusercontent.com/assets/507289/24883777/4814ca4a-1dfb-11e7-9750-6d43fb6ad4f3.png)

When set to "Flex end", the items are aligned to the top, as expected:
![end-fits](https://cloud.githubusercontent.com/assets/507289/24883787/4e255468-1dfb-11e7-9fb9-6e04ee4135d1.png)

However, when there are more items than can fit, I was expecting the items towards the *bottom* to be clipped, since that would mirror the behaviour from before. Surprisingly, it's still the items towards the top that are clipped:
![end-doesnot-fit](https://cloud.githubusercontent.com/assets/507289/24883792/5590e7d0-1dfb-11e7-88b4-52bedbff9de0.png)

To put it another way: when the items fit within the flexbox, "flex start" and "flex end" exhibit opposite behaviour, but when the items do not fit, they both exhibit the same behaviour.

Is this intended behaviour? If so, is there a way to specify that wrap reverse should clip earlier items rather than later items?

**Comment by thagikura** [comment]
Thanks for reporting this!

Looks like this is not an intended behavior when the size of the content is larger than the flex container and alignContent is set to flex_end.
Let me mark as a bug and I'll fix it on the master branch.

**Comment by soumya92** [comment]
Awesome! Thanks for the quick turnaround 👍
